### ISSUE TITLE
can't login and there's no other menu

### ISSUE THREAD

**Issue by goodknife** [AlexanderSharykin/CardIdleRemastered#5]
yesterday it works fine but today when i'm start program i see only two option (no queue and another tabs) so i thought it's a bug and delete the Settings.txt from AppData\Local\CardIdleRemastered

but now i can't login again (see image below), even tho i already input my username, password, and steam authen code correctly (i already check AppData\Local\CardIdleRemastered and found Settings.txt contain what seems to be my steam id data)

image here: https://cdn.discordapp.com/attachments/334511271275921409/372702921047146512/unknown.png

**Comment by AlexanderSharykin** [comment]
thank you for report. haven't launched Card Idle for some time myself, can't say when it started
bad new: **issue confirmed**, good news: I found the key problem (profile page  html structure was changed) and working on a hot fix
tech_details :
{
https://github.com/AlexanderSharykin/CardIdleRemastered/blob/master/SourceCode/CardIdleRemastered/SteamParser.cs#L32
page layout was changed!
link `a[@class=\"global_action_link\"]"` exists in both version (with or without login)
}

to clarify about your situation
> when i'm start program i see only two option (no queue and another tabs)
Card Idle tried to verify that your session is now expired (by connecting to Steam and reading profile page)
Your session probably was ok, but verification method bugged and Card Idle didn't do anything that requires login - it didn't load badges - and did show only free to use functionality

> delete the Settings.txt from AppData\Local\CardIdleRemastered
unless Settings.txt is corrupted, you don't need to delete it. if Card Idle doesn't show badges, try login again using (login) link on main screen (at the moment wait for a fix)
